test/specs/components/RadioGroup/RadioGroup-test.tsx

```diff
@@ -102,29 +108,30 @@ describe('RadioGroup', () => {
     })
 
     describe('click event handler', () => {
-      it('should set the value when item is clicked', () => {
-        const checkedValueChanged = jest.fn()
-        const wrapper = mountWithProvider(
-          <RadioGroup items={getItems()} checkedValueChanged={checkedValueChanged} />,
-        )
-        const radioGroupItems = wrapper.find('RadioGroupItem')
+      !isChildrenApiTest &&
```

Comment:
Please, use `if` in such cases it will be more clear. Also, I saw that you're using in both cases `!isChildrenApiTest` may be it should `isShorthandApiTest`?

Reply:
I used that parameter name in order to emphasize the tests are not supported by children API, i thought it's more readable. I can still make the change if you're feeling strongly for it @layershifter

Reply:
Yes, let's change it. Because in both cases you're using negative condition.

Reply:
done as suggested